lib/surface_bulma/dropdown.ex

```diff
@@ -0,0 +1,62 @@
+defmodule SurfaceBulma.Dropdown do
+  use Surface.LiveComponent
+
+  @moduledoc """
+  An interactive dropdown menu for discoverable content.
+
+  [Bulma Docs - Dropdown](https://bulma.io/documentation/components/dropdown/)
+  """
+
+  @doc "Is always open"
+  prop is_active, :boolean
```

Comment:
For consistency, maybe this could be renamed to `active`, as this is used in other places.